AirCasting/SessionViews/SessionCellView.swift

```diff
@@ -46,7 +49,7 @@ struct SessionCellView: View {
         let allLocationPoints = provider.allLocations
         let points = allLocationPoints.map { (location) in
             PathPoint(location: location.coordinate,
-                      measurement: Float(arc4random() % 200))
+                      measurement: Float(-10))
```

Comment:
what does `-10` mean? Seams a little magical. please make a variable and "define" this magic. 
![tenor](https://user-images.githubusercontent.com/1284783/117362047-15099900-aebb-11eb-9609-be3392827436.gif)

Reply:
It's a magical way to avoid crashing the app because random measurements that were used for map view were bigger than thresholds. It was necessary here because I needed to check if the slider  & HeatMap work properly on both the graph and map view. 
I know, it's not a real fix - why this shortcut? Because it'll be fixed when we start providing a map with real data, and in this case, there's not much difference between generating random integers and setting a fixed value. 
But it's my bad, I should have added a comment there.